example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsZIOApp.scala

```diff
@@ -0,0 +1,56 @@
+package xs4s.example
+
+import zio._
+import zio.console._
+import zio.stream._
+import zio.blocking.Blocking
+import java.io.IOException
+import java.io.InputStream
+import java.net.URL
+import java.net.URLConnection
+import java.util.zip.GZIPInputStream
+import java.util.zip.InflaterInputStream
+import java.util.zip.Inflater
+import scala.collection.JavaConverters._
+import xs4s.ziocompat._
+import xs4s.syntax.zio._
+
+object FindMostPopularWikipediaKeywordsZIOApp extends zio.App {
+
+  def run(args: List[String]): URIO[ZEnv, ExitCode] = {
+
+    wikipediaXmlBytes
+      .via(byteStreamToXmlEventStream()(_))
```

Comment:
might make sense to expand that `(_)` part so it looks like into `.via(somethingMeaningfulForANewbie => byteStream(..)(somethingMeaningful...)`. I'm a newbie with ZIO so it'd be helpful here